src/workersrc.js

```diff
@@ -48,86 +45,7 @@ function initializeCustomPDFViewerApplication() {
 	PDFViewerApplicationOptions.set('cMapUrl', document.getElementsByTagName('head')[0].getAttribute('data-cmapurl'))
 	PDFViewerApplicationOptions.set('enablePermissions', true)
 
-	console.debug('Initialized files_pdfviewer', PDFViewerApplicationOptions.getAll())
-
-	// The download has to be forced to use the URL of the file; by default
-	// "PDFViewerApplication.download" uses a blob, but this causes a CSP error
-	// (at least, in Firefox) when trying to download it.
-	PDFViewerApplication.download = function() {
```

Comment:
Download was already disabled within the app (we have the big button on top)

Reply:
https://github.com/nextcloud/files_pdfviewer/blob/bbf6b54fcccf0719d3b4005a3422a473534570ea/templates/viewer.php#L144